pages/web/hosting/hosting_migrating_to_ovh/guide.pt-pt.md

```diff
@@ -1,174 +1,258 @@
-|5|Validar a transferência|Este processo varia conforme a extensão do domínio. Por vezes, é necessário validar a transferência. Normalmente, o pedido de validação é enviado por e-mail. Siga os passos indicados para validar a transferência. |
+Para migrar o conteúdo dos endereços antigos, existem duas opções.
 
-#### 2 - Transferir o site
+**Opção 1**: utilize a nossa ferramenta [OVHcloud Mail Migrator (OMM)](https://omm.ovh.net/) {.external} que permite copiar o conteúdo dos endereços de e-mail registados no seu antigo fornecedor para os criados na OVHcloud. Para mais informações, consulte o nosso manual "[Migrar contas de e-mail via OVH Mail Migrator](https://docs.ovh.com/pt/microsoft-collaborative-solutions/exchange-migracao-de-contas-email-ovh-mail-migrator/)".
 
-|Passos|Descrição|Detalhes|
-|---|---|---|
-|1|Efetuar um backup (cópia) do site|Faça o backup de todos os ficheiros do site e, se necessário, da base de dados. O backup completo é essencial para garantir o sucesso da transferência do site para a OVH.|
-|2|Colocar o seu site online com o serviços OVH|Aceda ao espaço de alojamento (FTP), para importar os ficheiros do site. Para ficarem online, os ficheiros devem ser colocados na pasta **"www"**. Os dados de acesso FTP são transmitidos por e-mail.|
-|3|Criação de uma base de dados OVH|Se o seu site funcionar com uma base de dados, terá de [criar uma nova base de dados](https://docs.ovh.com/pt/hosting/gestao-de-uma-base-de-dados-a-partir-de-um-alojamento-partilhado-ovh/){.external} na [Área de Cliente OVH](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.pt/&ovhSubsidiary=pt){.external}.|
-|4|Importar dados existentes na base de dados inicial|Importe o backup da base de dados usando [a ferramenta disponível na Área de Cliente](https://docs.ovh.com/pt/hosting/partilhado_guia_de_importacao_de_uma_base_de_dados_mysql/){.external}.|
-|5|Associar o site à nova base de dados|As informações da base de dados inicial estão guardadas no ficheiro de configuração do seu site. No espaço de armazenamento OVH, altere o ficheiro, introduzindo as informações relativas à base de dados OVH.|
+Sugerimos que não utilize o `Tipo de servidor`{.action} **POP** na parte `Conta source`{.action}. Este protocolo apaga os e-mails do seu antigo servidor para os enviar para o servidor OVHcloud de destino. Neste caso, já não poderá comparar o conteúdo do antigo e do novo endereço de e-mail.
 
-A configuração do domínio continua inalterada. Se a resolução DNS ainda estiver ativa, o seu site continua a ser apresentado através do alojamento do operador anterior.
+Para a parte `Conta de destino`{.action}, introduza apenas o endereço de e-mail OVHcloud em questão e a respetiva palavra-passe. Isto permitindo o `Type de servidor`{.action} em `Hosted by OVH (Autodetect)`{.action}.
```

Comment:
Please check that `Type de servidor`{.action} is the correct PT wording

Reply:
tipo de serviço